### ISSUE TITLE
Relative positioning

### ISSUE THREAD

**Issue by ob7** [pzuraq/liquid-wormhole#12]
I was using ember-wormhole to insert a component in my application.hbs from the index.hbs view.

It works as expected, removing whatever content is in the target ID and adding new content to it.

I thought liquid wormhole would work the same, only animate it.

My problem is, liquid wormhole wont insert the wormhole into an element already created within my application.hbs, instead it positions itself absolute above the whole dom.

How can I have liquid-wormhole render into another element already created rather than making a newly positioned element absolute over the whole dom?  I need this to work the same way ember-wormhole works in that sense, where I can insert the wormhole anywhere in the main application ember view rather than outside it in an entirely seperate view.

Thanks, I hope its possible!

**Comment by pzuraq** [comment]
So, when I first started working on `liquid-tether` I was trying to implement it in the same way that `ember-tether` works, which is using `ember-wormhole` under the hood. `ember-wormhole` is remarkably simple - just give it the selector and it moves it wherever you want.

Unfortunately, `liquid-fire` does a _lot_ under the hood and it can't be done this easily. I tried several attempts at replicating the `ember-wormhole` API, but it wasn't successful.

A little backstory - when I first started this project, I looked around at the various use cases for `ember-wormhole` and `ember-tether` to see what people were using it for. There were basically two use cases:

1. People were using it to warp elements to the top of the DOM for things like modals, popovers, tooltips, etc.
2. People were using it to warp elements elsewhere in the DOM, usually because they were working with a legacy application that was not entirely Ember JS based, and had to inject components into the legacy app.

I decided that the more valuable use case to solve for was the first, as newer Ember apps were the priority and legacy applications should not be holding back the Ember community. So, I narrowed the `liquid-wormhole` API to manage only elements at the top of the DOM.

That's the history behind the decision, and why this forked off from `ember-wormhole` in the first place.

Now, I'm curious what your use case is for. It doesn't seem like you're injecting components into a legacy app, because you mention that the element is already in `application.hbs`. Can you not just style the `liquid-wormhole` outlets the way your current element is styled?

As for adding this functionality and bringing the `liquid-wormhole` API into alignment with the `ember-wormhole` API, it may be possible. I'm hesitant to do it, if only because I think its an anti-pattern (i.e. I don't think people should really be warping things around your app in an uncontrolled way, it's going to cause problems like the one you're having right now), but if it can't be handled any other way I'm willing to take a look at it and figure out a solution.

**Comment by pzuraq** [comment]
Screenshots are infinitely helpful in understanding UI components and their needs, by the way 😉

**Comment by ob7** [comment]
Basically the website I'm building has a slideshow in the application.hbs view, as its on every page, so I used ember-wormhole to update it based on the current route, as the header slideshow changes per route. But in the end I ended up putting the slideshow in each router because it looks nice just animating the entire page including slideshow to the left or right whenever the route changes, instead of animating the lower page with liquid-fire and then the slideshow at the top doing something different.  

Thanks for your response, I figured it was so.  Still, it would be a sweet feature to animate ember-wormholes no matter where they are in the DOM.  In this case it was just a slideshow and I ended up moving it into each router instead of staying in the application view.
